central/debug/service/service.go

```diff
@@ -735,6 +740,23 @@ func (s *serviceImpl) writeZippedDebugDump(ctx context.Context, w http.ResponseW
 			return writeTelemetryData(zipWriter, telemetryData)
 		})
 	}
+	diagBundleTasks.Go(func(ctx context.Context) error {
+		config, err := s.getDeleRegConfigs(ctx)
+
+		var b []byte
+		if err != nil {
+			b, _ = json.Marshal(map[string]string{
+				"message": err.Error(),
+			})
+		} else {
+			b, err = protojson.MarshalOptions{Multiline: true}.Marshal(config.(proto.Message))
```

Comment:
Can avoid the type assertion here if change the return from `getDeleRegConfigs` to `(proto.Message, error)`